api/api/settings.py

```diff
@@ -76,10 +79,10 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'postgres',
```

Comment:
Since this stuff if specific to your dev environment, can you comment this and leave the configuration as is so that it doesn't break other peoples' docker configurations?